pkg/kubelet/server.go

```diff
@@ -148,6 +149,8 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.mux.HandleFunc("/logs/", s.handleLogs)
 	s.mux.HandleFunc("/containerLogs/", s.handleContainerLogs)
 	s.mux.Handle("/metrics", prometheus.Handler())
+	// The /runningpods endpoint is used for testing only.
+	s.mux.HandleFunc("/runningpods", s.handleRunningPods)
```

Comment:
Hmm...I missed this comment. I could do a follow-up PR to make the names consistent, whichever format we decide to use. What do you think?